sdk/helper/ldaputil/config.go

```diff
@@ -239,6 +239,12 @@ Default: ({{.UserAttr}}={{.Username}})`,
 			Default:     "90s",
 		},
 
+		"connection_timeout": {
+			Type:        framework.TypeDurationSecond,
+			Description: "Timeout, in seconds, when attempting to connect to the LDAP server before trying the next URL in the configuration.",
+			Default:     "60s",
```

Comment:
Wondering if we should change this to be lower than the Vault client timeout to avoid errors in failovers without having to adjust this. If not, the description should include a blurb on lowering this if multiple servers are specified in the `url` to avoid failover errors.

Reply:
I was hesitant to change the default connection timeouts (60s) in the LDAP package but its an awfully long timeout duration. I think 30s is much more appropriate as a default IMO.